charts/refractr/templates/issuer.yaml

```diff
@@ -18,4 +18,20 @@ spec:
       - http01:
           ingress:
             class: nginx
+      - dns01:
+          # This is so we can instruct LE to follow a CNAME on the target domain to
+          # a domain we control (refractr.mozit.cloud) in route53
+          cnameStrategy: Follow
+          route53:
+            region: us-west-2
+            hostedZoneID: Z06469063HC60S7SHF706
```

Comment:
Why hardcode this value here when you can templatize it? You should make this into a template